birdhouse/scripts/get-components-json.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
This is odd, why is this file here?  It's part of the other PR already merged!

Reply:
Oh, left over item prior to rename (I cherry-picked it). Will remove.